City-of-Helsinki/paatokset-search
# PP-245: Add translation capability with i18n + reorder files

Added internationalization capability, reordered some files for a better file structure.

To test: 
- Run `npm i; npm start`
- Open the app at '/localhost:3000' or network specific address
- Try adding a language parameter to the url, eg. 'localhost:3000/fi' or '/en'. Form texts should change correspondingly between fi / en translations
